iree/samples/custom_modules/dialect/test/conversion.mlir

```diff
@@ -54,6 +54,22 @@ func @dynamicTensorToMessage(%arg0 : tensor<?x?xf32>, %arg1 : index, %arg2 : ind
 
 // -----
 
+// CHECK-LABEL: @dynamicTensorToMessage2
+func @dynamicTensorToMessage2(%arg0 : tensor<?x?xf32>, %arg1: !shapex.ranked_shape<[?, ?]> {iree.reflection = {}}) {
+  // CHECK-DAG: [[TYPE:%.+]] = vm.const.i32 50331680 : i32
```

Comment:
The convention is not to capture `%`, i.e., use `%[[TYPE:.+]]` .

Reply:
Some of these files/cases have done the opposite since inception. Usually for shall changes, I follow the convention of the surrounding test.